src/gordon_gcp/plugins/janitor/authority.py

```diff
@@ -174,8 +174,15 @@ def _filter_results(self, results):
             coros.add(coro)
 
         all_results = await asyncio.gather(*coros, return_exceptions=True)
-
-        return self._filter_results(all_results)
+        all_results = self._filter_results(all_results)
+
+        instances = []
+        for instance in all_results:
+            if instance['status'] == 'TERMINATED':
+                logging.info(f'Skipping terminated instance: "{instance}"')
+                continue
+            instances.append(instance)
```

Comment:
Any harm in moving this up to the `_filter_results` method? We seem to be doing some skipping up there.

Reply:
I was thinking this is better grouping of functionality - filter results filters the results themselves, get instances gets (filtered) instances?